lib/std/crypto/blake3.zig

```diff
@@ -666,6 +681,95 @@ fn leftSubtreeLen(input_len: usize) usize {
     return @intCast(roundDownToPowerOf2(full_chunks) * chunk_length);
 }
 
+const ChunkBatchContext = struct {
+    input: []const u8,
+    start_chunk: usize,
+    end_chunk: usize,
+    cvs: [][8]u32,
+    key: [8]u32,
+    flags: Flags,
+};
+
+fn processChunkBatch(ctx: ChunkBatchContext) void {
```

Comment:
```suggestion
fn process(ctx: ChunkBatchContext) void {
```

(and move it inside `ChunkBatch` namespace)